### PR TITLE
Deleting google sheet data before writing new data

### PR DESCRIPTION
**Pull request by etiksha**
Modifying gsheet.py to deleting data from sheet before writing new data
Modifying vm_metrics.py to return the metrics data instead of writing into google sheet so that the data from all tests can be combined together and then written into the google sheet through fetch_metrics.py.

### CODE REVIEW

File: perfmetrics/scripts/vm_metrics/vm_metrics.py [hunk 1]
```diff
@@ -295,8 +313,7 @@ def main() -> None:
   test_type = sys.argv[5]
   worksheet_name = sys.argv[6]
   vm_metrics = VmMetrics()
-  vm_metrics.fetch_metrics_and_write_to_google_sheet(start_time_sec, end_time_sec, instance,
-                                                     period, test_type, worksheet_name)
+  vm_metrics.fetch_metrics_and_write_to_google_sheet(start_time_sec, end_time_sec, instance, period, test_type, worksheet_name)
```
Comment (avidullu) [inline perfmetrics/scripts/vm_metrics/vm_metrics.py:316]: Can you preserve the previous formatting here and everywhere (<80 chars per line)

File: perfmetrics/scripts/gsheet/gsheet.py [hunk 2]
```diff
@@ -12,6 +12,7 @@ def _get_sheets_service_client():
   service = build('sheets', 'v4', credentials=creds)
   return service
```
Comment (avidullu) [inline perfmetrics/scripts/gsheet/gsheet.py:14]: Can you write a comment here about how this file works?

File: perfmetrics/scripts/gsheet/gsheet.py [hunk 3]
```diff
@@ -12,6 +12,7 @@ def _get_sheets_service_client():
...
 
+
 def write_to_google_sheet(worksheet: str, data) -> None:
   """Calls the API to update the values of a sheet.
```
Comment (avidullu) [inline perfmetrics/scripts/gsheet/gsheet.py:17]: Fix these comments here and other functions too

**Review by avidullu — APPROVED** [review]

File: perfmetrics/scripts/vm_metrics/vm_metrics.py (hunk 1 above)
Comment (etiksha) [reply perfmetrics/scripts/vm_metrics/vm_metrics.py]: done

File: perfmetrics/scripts/gsheet/gsheet.py (hunk 2 above)
Comment (etiksha) [reply perfmetrics/scripts/gsheet/gsheet.py:18]: done

File: perfmetrics/scripts/gsheet/gsheet.py (hunk 3 above)
Comment (etiksha) [reply perfmetrics/scripts/gsheet/gsheet.py]: done

**Pull request merged**